From b6fd8296d751792986bb13d1edf16af63712592a Mon Sep 17 00:00:00 2001
From: Mathieu Perrier <mathieux.perrier@intel.com>
Date: Wed, 7 Aug 2013 10:38:40 +0200
Subject: Cannot use a vnd_ctp7160.txt configuration file for BT

BZ: 129034

The configuration file vnd_redhookbay.txt is used for BT, whereas
the BT component used is the same one as the victoriabay. (BC4334).

This patch modifies the conf file loaded at BT initializaion for
ctp7160.

Change-Id: I362c3c538c9d8db1c1319790c3d54bc3115fb2c6
Signed-off-by: Mathieu Perrier <mathieux.perrier@intel.com>
Reviewed-on: http://android.intel.com:8080/126153
Reviewed-by: Dixon, CharlesX <charlesx.dixon@intel.com>
Tested-by: Dixon, CharlesX <charlesx.dixon@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 conf/intel/ctp7160/bt_vendor.conf |   19 +++++++++++++++++++
 1 files changed, 19 insertions(+), 0 deletions(-)

diff --git a/conf/intel/ctp7160/bt_vendor.conf b/conf/intel/ctp7160/bt_vendor.conf
index 93aa23c..b06e977 100644
--- a/conf/intel/ctp7160/bt_vendor.conf
+++ b/conf/intel/ctp7160/bt_vendor.conf
@@ -3,3 +3,22 @@ UartPort = /dev/ttyMFD0
 
 # Firmware patch file location
 FwPatchFileRootPath = /etc/firmware/bt/
+LPM_IDLE_TIMEOUT_MULTIPLE = 5
+SCO_USE_I2S_INTERFACE = FALSE
+SCO_I2SPCM_IF_ROLE = 1
+SCO_I2SPCM_IF_SAMPLE_RATE = 0
+SCO_I2SPCM_IF_CLOCK_RATE = 0
+SCO_PCM_IF_CLOCK_RATE = 0
+SCO_PCM_IF_FRAME_TYPE = 1
+SCO_PCM_IF_SYNC_MODE = 1
+SCO_PCM_IF_CLOCK_MODE = 1
+PCM_DATA_FMT_SHIFT_MODE = 0
+PCM_DATA_FMT_FILL_BITS = 0
+PCM_DATA_FMT_FILL_METHOD = 3
+PCM_DATA_FMT_FILL_NUM = 3
+PCM_DATA_FMT_JUSTIFY_MODE = 0
+DEPLOY_4335A_AXI_BRIDGE_PATCH = TRUE
+BTVND_DBG = FALSE
+BTHW_DBG = FALSE
+VNDUSERIAL_DBG = FALSE
+UPIO_DBG = FALSE
-- 
1.7.4.1

